tests/ImageTests.php

```diff
@@ -447,7 +460,12 @@ protected function output($file)
     public function setUp()
     {
         $dir = $this->output('');
-        `rm -rf $dir`;
+        
+        if (PHP_OS === 'WINNT') {
+            `rd /s /q "$dir"`;
```

Comment:
Symfony FileSystem might be too much just fr one function. I found this package: https://github.com/perchten/php-rmrdir

Do you want me to make the change in this PR?